.github/workflows/ci-integration-tests-gcp.yml

```diff
@@ -397,7 +204,7 @@ jobs:
   test-full-sync:
```

Comment:
Can you please run a manual full sync on this PR before merging it?
We'll need to fix bug #7618 first.

(We forgot to run a full sync for the self-hosted runners change, and so I'm currently checking all the PRs back past that change as part of diagnosing bug #7618.)